### ISSUE TITLE
Установка ПО

### ISSUE THREAD

**Issue by Kim134** [MantisReligiosa/Display-control#46]

**Comment by Kim134** [comment]
Алексей, думаю нужно подготовить файлы для установки базы данных, и тоже написать инструкцию по установке.
Мы не сможем всегда вовремя реагировать на тех поддержку, а это негатив.
Т.е. нужен полные установочные пакеты для развертывания ПО, а так же продумать и сделать установку всего - легкую, что бы с установкой и запуском ПО справился обычный пользователь.

**Comment by MantisReligiosa** [comment]
У меня сомнения - стоит ли чинить процесс установки лишь только для облегчения установки пользователем. Пока у нас нет защиты, вот эти танцы с бубном защищают нас от копирования

**Comment by MantisReligiosa** [comment]
Добавлено начина с версии 0.0.6 
Ссылка на zip: https://dev.azure.com/mantis-religiosa/f885b45e-6407-4363-a4b4-3c2f41dda368/_apis/build/builds/59/artifacts?artifactName=Archive&fileId=D4A8245355B6A254B4790A7A1CE4658136C4E871A2B5FB9FDACC22642AED87A502&fileName=Display_control.zip&api-version=5.0-preview.3
Ссылка на msi: https://dev.azure.com/mantis-religiosa/f885b45e-6407-4363-a4b4-3c2f41dda368/_apis/build/builds/59/artifacts?artifactName=Installer&fileId=551D5377BB17F5B959CA1FEF77A67AC40531D3846D62E978BA5FF2C27CCD7C9A02&fileName=Display_control.msi&api-version=5.0-preview.3

**Comment by MantisReligiosa** [comment]
Проблема при разворачивании на тестовом ноуте

**Comment by Kim134** [comment]
Здесь вопрос такой: нужно ли будет чистить постоянно базу?

**Comment by MantisReligiosa** [comment]
Нет, не нужно. Это всё последствия серьёзной переработки базы при реализации мнемосхем. Теперь уже все ошибки исправлены и базу чистить не придется

**Comment by Kim134** [comment]
ОТЛИЧНО!
ты меня успокоил! я подумал нужно будет заказчику при каждом обновлении чистить.
